docs/compose-file.md

```diff
@@ -932,6 +948,22 @@ refer to it within the Compose file:
         external:
           name: actual-name-of-network
 
+### labels
+
```

Comment:
Needs the `> Added in version 2.1 file format` line, similar to [this](https://github.com/docker/compose/blob/master/docs/compose-file.md#link_local_ips)